snuba/datasets/cdc/types.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Literal, Sequence, Union, TypedDict
+
+
+class BeginEvent(TypedDict):
+    event: Literal["begin"]
+    xid: int
+
+
+class ChangeEvent(TypedDict):
+    event: Literal["change"]
+    xid: int
+    timestamp: str
```

Comment:
I'm not sure if this inclusion of the timestamp is actually correct. It depends on whether `include-timestamp` is provided to our production wal2json configuration. It was present in the groupassignee test fixtures, but not in the groupedmessage ones.

If it is not included in the configuration, we should consider adding it in case we want to merge streams on timestamp in the future.

Reply:
It is set to True in our prod config. The only reason it was not in the groupedmessage test is that we added it after groupedmessage and before creatinng groupassignee.